PM-1064 Deploy MR1 features in the SUD app

andriy-morgunov-epam opened the pull request:
https://1mojio.atlassian.net/browse/PM-1064

⁉️ What does this PR do?

- Appended new properties that Platform supports for Vehicle

Seatbelt
Seats
BrakeFluid
NextService
HybridBattery
HybridEngine

📖 Any background context you want to provide?

- None

andriy-morgunov-epam committed:
PHIOS-6725 [Tech Debt] iOS - Deleting multiple virtual boundaries does not delete all boundaries

https://1mojio.atlassian.net/browse/PHIOS-6725

⁉️ What does this PR do?

- Appended new properties that Platform supports for Vehicle

Seatbelt
Seats
BrakeFluid
NextService
HybridBattery
HybridEngine

📖 Any background context you want to provide?

- None

andriy-morgunov-epam committed:
PM-1064 Deploy MR1 features in the SUD app

https://1mojio.atlassian.net/browse/PM-1064

⁉️ What does this PR do?

- Fixed parsing HybridBattery

📖 Any background context you want to provide?

- None

pauluhn commented on MojioSDK.xcodeproj/project.pbxproj:
```diff
@@ -603,7 +621,7 @@
 				BC879B581CBEFFDD00F9177B /* RPM.swift */,
 				BC879B5A1CBEFFDD00F9177B /* Speed.swift */,
 				BC879B4A1CBEFFDD00F9177B /* TimePeriod.swift */,
-				37BC6C8223129E4B0054B420 /* EngineOilTemperature.swift */,
```
#? intended?

andriy-morgunov-epam replied:
@pauluhn This is Xcode project file editing side effect. In line 614 you can see the new line for "EngineOilTemperature". So this line changed the position, it wasn't deleted

pauluhn commented on MojioSDK/Models/Vehicles/BrakeFluid.swift:
```diff
@@ -0,0 +1,67 @@
+/******************************************************************************
+ * Moj.io Inc. CONFIDENTIAL
+ * 2017 Copyright Moj.io Inc.
```
#req update copyright

andriy-morgunov-epam replied:
@pauluhn Copyrights were updated

andriy-morgunov-epam committed:
PHIOS-6873 Enable brake fluid monitoring in the app

https://1mojio.atlassian.net/browse/PHIOS-6873

⁉️ What does this PR do?

- Appended new type to ActivityType
- Updated copyrights

📖 Any background context you want to provide?

None

andriimakukha commented:
@andriy-morgunov-epam Copyrights were updated in scope https://github.com/mojio/mojio-ios-sdk/pull/342.

andriy-morgunov-epam committed:
Revert "PHIOS-6873 Enable brake fluid monitoring in the app"

This reverts commit 735f230929e6c0859b39c6d83b1bb43a4e5ebae3.

andriy-morgunov-epam committed:
PHIOS-6873 Enable brake fluid monitoring in the app

https://1mojio.atlassian.net/browse/PHIOS-6873

⁉️ What does this PR do?

- Appended new type to ActivityType

📖 Any background context you want to provide?

None

andriy-morgunov-epam committed:
Revert "Revert "PHIOS-6873 Enable brake fluid monitoring in the app""

This reverts commit 68f2ee1354d6c973a84ba34283e0b2fc6ad49bda.

andriy-morgunov-epam committed:
PHIOS-6874 Enable smart maintenance for the vehicle, where supported

https://1mojio.atlassian.net/browse/PHIOS-6874

⁉️ What does this PR do?

- Appended new type to ActivityType

📖 Any background context you want to provide?

None

andriy-morgunov-epam committed:
PHIOS-6874 Enable smart maintenance for the vehicle, where supported
PHIOS-6873 Enable brake fluid monitoring in the app

https://1mojio.atlassian.net/browse/PHIOS-6874
https://1mojio.atlassian.net/browse/PHIOS-6873

⁉️ What does this PR do?

- Appended new types to TimelineActivityType

📖 Any background context you want to provide?

None
